assets/app/views/create/fromimage.html

```diff
@@ -0,0 +1,227 @@
+                  </div>
+                  <div>
+                    <label>Deployment configuration changes: </label>
+                    <span>{{deploymentConfig.deployOnConfigChange | yesNo}}</span>
+                  </div>
+                  <h3>Environment Variables <span class="help action-inline">
+                    <a href data-toggle="tooltip" data-placement="right" 
+                       data-original-title="Environment variables are used to configure and pass information to images.">
+                      <i class="pficon pficon-help"></i> 
+                    </a>
+                  </span></h3>
+                  
+                  <osc-key-values
+                    entries="deploymentConfig.envVars"
+                    delimiter="="
+                    key-validator="env"
+                    delete-policy="added"
+                    editable="false"
+                   />
+                </div>
+                <div class="animate-drawer" ng-show="$parent.expand">
+                  <h3>Autodeploy when</h3>
+                  <div>
+                    <label class="checkbox">
+                      <input type="checkbox" ng-model="deploymentConfig.deployOnNewImage">
+                      New image is available
+                    </label>
+                  </div>
+                  <div>
+                  <label class="checkbox">
+                    <input type="checkbox" ng-model="deploymentConfig.deployOnConfigChange">
+                    Deployment configuration changes
+                  </label>
+                  </div>
+                  <div>
+                    <h3>Environment Variables <span class="help action-inline">
+                      <a href data-toggle="tooltip" data-placement="right" 
+                         data-original-title="Environment variables are used to configure and pass information to images.">
+                        <i class="pficon pficon-help"></i> 
+                      </a>
+                    </span></h3>
+                    <osc-key-values
+                      entries="deploymentConfig.envVars"
+                      delimiter="="
+                      key-validator="env"
+                      delete-policy="added"
+                      key-validation-tooltip="A valid environment variable name is an alphanumeric (a-z and 0-9) string beginning with a letter that may contain underscores."
+                    />
+                  </div>
+                </div>
+              </osc-form-section>
+
+              <!-- /deployment config -->
+
+              <osc-form-section 
+                header="Build Configuration"
+                about-title="Build Configuration"
+                about="A build configuration describes how to build your deployable image which includes
+                  your source, the image upon which to base the final image, and when to launch new builds."
```

Comment:
instead of "the image upon which to base the final image"  maybe "the base image to build with"